packages/client/modules/teamDashboard/components/ProviderRow/MattermostPanel.tsx

```diff
@@ -68,8 +69,12 @@ const MattermostPanel = (props: Props) => {
         teamMember(teamId: $teamId) {
           integrations {
             mattermost {
-              isActive
-              webhookUrl
+              auth {
+                provider {
+                  id
+                  webhookUrl
+                }
+              }
```

Comment:
-2, there's no `auth` for webhook providers added currently, this will always be empty currently. as a result, UI won't show active integration